pom.xml

```diff
@@ -95,6 +95,7 @@
 
         <!-- experimental -->
         <module>experimental/langchain4j-experimental-sql</module>
+        <module>langchain4j-scoring</module>
 
     </modules>
 
```

Comment:
would be great to add documentation page similar to `docs/docs/integrations/scoring-reranking-models/2-cohere.md`

Reply:
I will add this